### ISSUE TITLE
Migrate wiki to MkDocs

### ISSUE THREAD

**Issue by seancolsen** [centerofci/mathesar#3079]
Motivated by [1 hour quick stab at migration to mkdocs](https://github.com/centerofci/mathesar-wiki/pull/99)

**Comment by seancolsen** [comment]
@kgodey,

One thing that I think is worth discussing ahead of time is the Japanese translation. I think our translation efforts are much better spent on the product (once we have the i18n system set up of course), and then secondarily the user-facing documentation. Maintaining a multilingual wiki does not seem worth the effort to me. I see the wiki's role as a resource for Mathesar developers. Given that all our development workflow happens in English, it seems appropriate to keep the wiki limited to English. During my 1 hour quick stab, I tried briefly to enable English and Japanese, but the multilingual setup didn't seem play well with the [mkdocs-awesome-pages](https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin) plugin (which I think is fairly important for the wiki).

Would you be okay with removing the Japanese translated content?

**Comment by kgodey** [comment]
@seancolsen Yes, that's fine. The content pre-dates the existence of mathesar.org (i.e. the wiki was our main website), and I think our translation efforts should go into mathesar.org and docs.mathesar.org.

The content is also out of date with the English version, so I'm not sure how useful it is anyway.
